README.md

```diff
@@ -18,7 +18,10 @@ Our team was tasked with designing and creating a functioning e-commerce web app
 ### Daniel Ghaly [LinkedIn] [GitHub]
 
 ### Related Products
-### Kavita Poduri
+### Kavita Poduri | [LinkedIn](https://www.linkedin.com/in/kvpoduri/) | [GitHub](https://github.com/yrgirlkv)
```

Comment:
Consider using badges to make your LinkedIn and Github links look cleaner. Can look at my markup as an example.